lldb/source/API/SBDebugger.cpp

```diff
@@ -702,6 +702,14 @@ SBStructuredData SBDebugger::GetBuildConfiguration() {
       "A boolean value that indicates if lua support is enabled in LLDB");
   AddLLVMTargets(*config_up);
 
+  bool swift_enabled = false;
```

Comment:
What happens if we wrap the AddBoolConfigEntry in the `ifdef` and don't set the entry when swift is disabled? If we upstream SWIFT it would be similar to `LLDB_ENABLE_PYTHON ` and we could just remove the ifdef wrapped around it.

Reply:
That seems to also do the trick (even though I feel like looking up a config value that doesn't exist should maybe be a hard error? Otherwise having typos in the setting string will just silently disable the thing).